drupal-org.make

```diff
@@ -72,12 +77,17 @@ projects[token][version] = 1.0
 projects[video_embed_field][type] = module
 projects[video_embed_field][version] = 1.5
 projects[views_infinite_scroll][type] = module
-projects[views_infinite_scroll][version] = 1.3
+projects[views_infinite_scroll][version] = 1.5
 projects[workbench_moderation][type] = module
 projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/2668006-2.patch
 projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/2847078-6.patch
 projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/workbench_moderation-quickedit-support-2749503-6.patch
 projects[workbench_moderation][version] = 1.2
+libraries[cropper][type] = library
```

Comment:
This library is required by Image Widget Crop and it's already on drupal.org's library packaging whitelist.